README.md

```diff
@@ -5,12 +5,36 @@
 ### Introduction
 
 The service-catalog project is in incubation to bring integration with service
-brokers to the Kubernetes ecosystem via the
-[Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker).
-A service broker is an endpoint that manages a set of services.  The end-goal of
-the service-catalog project is to provide a way for Kubernetes users to consume
-services from brokers and easily configure their applications to use those
-services.
+brokers to the Kubernetes ecosystem via the [Open Service Broker
+API](https://github.com/openservicebrokerapi/servicebroker). A service broker
+is an endpoint that manages a set of services.  The end-goal of the service-
```

Comment:
@chrislovecnm I appreciate the suggestions, but your text is not quite accurate in a couple ways:

1.  This is not a general application configuration tool - it is about consuming services only
2.  'the service broker' is not really the thing in play - this project allows users to consume broker-provided services